templates/custom_play.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'base.html' %}
+{% block head %}
+
+<script type="module" src="{{url_for('static', filename='js/custom_play.js') }}"></script>
+
+{% endblock %}
+
+
+{% block content %}
+
+<div id="app">
+
+    <h2> Choose Your Starting and Ending Article! </h2>
+    <hr>
+
+    <div class="col-md-4 col-sm-6">
+        <ac-input :text="start" @update:text="newValue => start = newValue" placeholder="Start Article"></ac-input>
```

Comment:
I think this is where you can do the `v-bind:start.sync` thing? It's just shorthand for this as far as I know.